### PR TITLE
assets/store_descriptions/en-US/strings.xml: add fullstop at end of sentences

### PR DESCRIPTION
**Pull request by dilyanpalauzov**
Closes https://github.com/openhab/openhab-android/issues/3631

### CODE REVIEW

File: assets/store_descriptions/en-US/strings.xml [hunk 1]
```diff
@@ -4,22 +4,22 @@
     <whatis>openHAB is a Java-based open-source home automation platform that integrates and combines a vast range of different smart home systems and technologies into one single solution.</whatis>
     <rules>On top of the unified abstraction layer all connected devices are available for the overarching automation rule engines and different user interfaces.</rules>
     <supported>Supported Products</supported>
-    <bindings>Over 200 specific add-ons provide support for brands, devices, technologies and communication protocols. Examples are Z-Wave, Philips Hue, Amazon Echo, Chromecast and Sonos. Discover all available add-ons and supported devices and/or functions at: https://www.openhab.org/addons/</bindings>
+    <bindings>Over 200 specific add-ons provide support for brands, devices, technologies and communication protocols. Examples are Z-Wave, Philips Hue, Amazon Echo, Chromecast and Sonos. Discover all available add-ons and supported devices and/or functions at: https://www.openhab.org/addons/.</bindings>
```
Comment (mueller-ma) [inline assets/store_descriptions/en-US/strings.xml:7]: I'd omit the full stop here...

File: assets/store_descriptions/en-US/strings.xml [hunk 2]
```diff
@@ -4,22 +4,22 @@
...
+    <about_foundation>The openHAB Foundation e.V. is a nonprofit organization with the mission to educate the public about the possibilities and benefits of free and open smart home solutions. Learn about the mission and the services of the foundation under https://www.openhabfoundation.org.</about_foundation>
     <important_note>Important note</important_note>
     <oh_server>You need an openHAB server for this app.</oh_server>
     <short_description>Vendor and technology agnostic open source home automation</short_description>
     <fdroid>The builds on F-Droid have FCM removed and therefore won't be able to receive push notifications from openHAB Cloud. Instead, notifications are polled on a regular basis in these builds.</fdroid>
     <fdroid_beta><![CDATA[You can install the beta version alongside the <a href="https://f-droid.org/packages/org.openhab.habdroid/">stable version</a>.]]></fdroid_beta>
-    <fdroid_privacy_policy>Privacy policy: https://www.openhabfoundation.org/privacy.html#android-app</fdroid_privacy_policy>
+    <fdroid_privacy_policy>Privacy policy: https://www.openhabfoundation.org/privacy.html#android-app.</fdroid_privacy_policy>
```
Comment (mueller-ma) [inline assets/store_descriptions/en-US/strings.xml:22]: ... and here

**Review by mueller-ma — CHANGES_REQUESTED** [review]
Please sign-off your commits with `git commit -s`

File: assets/store_descriptions/en-US/strings.xml (hunk 1 above)
Comment (dilyanpalauzov) [reply assets/store_descriptions/en-US/strings.xml]: > Please sign-off your commits with `git commit -s`

This is irrelevant.

**Pull request merged**